### DISCUSSION TITLE
Firmware Update Shelly Plug S does not work

### DISCUSSION THREAD

**Discussion by donnerhund** [arendst/Tasmota#15607] [Q&A] [answered]
Dear all,

I am currently trying to flash 2 Shelly Plug S sockets with the Tasmoto firmware.
I am proceeding as described in https://github.com/arendst/mgos-to-tasmota. But the update does not seem to start.
I have downloaded the ZIP file (mg2tasmota-ShellyPlugS.zip) directly to my computer and found that there is a file shelly-plug-u1-0.3.1\shelly-plug-u1.bin in the archive. For me as an absolute Tasmoto beginner, it seems that the archive for Shelly Plug S does not contain the correct files. Can someone confirm this and correct the possible error, or tell me how I can flash the Shelly with Tamoto (without opening the sockets).

Thanks a lot

**Comment by barbudor** [comment, marked as answer]
May be @arendst will be able to advise if \shelly-plug-u1.bin` is the right firmware

**Reply by arendst** [reply]
You might want to try the original site https://github.com/yaourdt/mgos-to-tasmota where the procedure is more elaborate but there the correct plug S files are present.

**Reply by donnerhund** [reply]
Thank you for pointing this out. I was now able to successfully flash to Tasmota.

Here are also the correct files in the ZIP file: http://dl.dasker.eu/firmware/mg2tasmota-ShellyPlugS.zip

**Comment by totheogreat** [comment]
Hello,

had the same issue. The answer from arendst solved it.

thx (me zen again :))
